lib/Transforms/transforms.bzl

```diff
@@ -0,0 +1,91 @@
+"""Macros to streamline the generation of HEIR transform tablegen-generated
+files."""
+
+load("@llvm-project//mlir:tblgen.bzl", "gentbl_cc_library")
+
+def add_heir_transforms(
+        pass_name = None,
```

Comment:
could you do me a huge favor to save time merging this, but can you add a `name` keyword argument to make buildifier happy? All macros need one (it can be unused and optional)

Reply:
Will do later tonight

Reply:
Done!